docs/spec/api.md

```diff
@@ -126,6 +126,13 @@ reference and shouldn't be used outside the specification other than to
 identify a set of modifications.
 
 <dl>
+  <dt>l</dt>
+  <dd>
+    <ul>
+      <li>Document TOOMANYREQUESTS error code.</li>
```

Comment:
This should have ended up in autogenerated error table, rather than on each request.